lib/runner.js

```diff
@@ -17,10 +16,8 @@ function hook(dirname, filename) {
 }
 
 function commandFor(hookName) {
-  const pkgFile = findup.sync(process.cwd(), 'package.json')
```

Comment:
nice to move this logic to somewhere isolated where it can be tested easier :+1: